website/docs/r/cis_tls_settings.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "ibm"
+page_title: "IBM: ibm_cis_tls_settings"
+sidebar_current: "docs-ibm-cis-tls-settings"
```

Comment:
sidebar content should match ibm.erb file
 please replace to `docs-ibm-resource-cis-tls-settings`

Reply:
replaced